packages/core/src/__tests__/simple-account.test.ts

```diff
@@ -58,11 +58,10 @@ describe("Simple Account Tests", () => {
   });
```

Comment:
We haven't really added any tests to ensure the targets and data of our batch calls are formatted as expected. Can we add there here or maybe in another test?